Gravity/Gravity/DAL/RSAPI/RsapiDao.Get.cs

```diff
@@ -118,13 +120,15 @@ internal void PopulateChildrenRecursively<T>(BaseDto baseDto, RDO objectRdo, Obj
 
 		private object GetChildObjectRecursively(BaseDto baseDto, RDO objectRdo, ObjectFieldsDepthLevel depthLevel, PropertyInfo property)
 		{
+			var fieldType = property.GetCustomAttribute<RelativityObjectFieldAttribute>()?.FieldType;
```

Comment:
I think this file can be reverted when we fix the attributes removed (as mentioned on GravityLevelOne.cs).